FreeOTP/FRNavigationController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  FRNavigationController.swift
+//  FreeOTP
+//
+//  Created by Vinícius Soares on 12/06/20.
+//  Copyright © 2020 Fedora Project. All rights reserved.
+//
+
+import UIKit
+
+class FRNavigationController: UINavigationController {
```

Comment:
What is the meaning of the naming choice `FRNavigationController` ?

Reply:
The first two letter of the project name. I wasn't very inspired that day haha

Reply:
I suppose `MainNavigationController` might be better, or something similar